### ISSUE TITLE
A11Y -- About dropdown not in the tab order

### ISSUE THREAD

**Issue by damiansian** [gutenbergtools/gutenbergsite#126]
The About dropdown picker is not in the keyboard focus path. 

![Keyboard tab order not showing the About picker in the tab order](https://github.com/user-attachments/assets/6fc1eb63-fde2-4da4-9ed0-6b31cef57d5d)

**Comment by eshellman** [comment]
This one should be straightforward @JohannesSeikowsky would you like to take this one? or I could do the autocat3 version and you could just copy

**Comment by JohannesSeikowsky** [comment]
I'm currently quite occupied with redoing the "Main Categories". So maybe better if you do the autocat3 version and I copy, as suggested.

**Comment by eshellman** [comment]
It's quite difficult to get the dropdown to be traversable. I can get the dropdown to display with some added css, but I think a little bit of javascript to activate the menu will be required.
